app/src/features/ReservationDetailsScreen/index.tsx

```diff
@@ -164,6 +169,11 @@ const ReservationDetailsScreen = () => {
                 <div>
                     <h2>Current price: {currentPrice}</h2>
                 </div>
+                {!currentUser && numberOfSelectedSeats > 0 &&(
+                    <div>
+                        <h3>You must be logged in to create reservations</h3>    
+                    </div>
+                )}
```

Comment:
Da li se dugme za rezervisanje vidi ako korisnik nije ulogovan? Ako da, mozes li da mu dodas uslov `currentUser &&`

Takodje, mozes ovde da sklonis uslov `numberOfSelectedSeats > 0`. Neka se poruka uvek vidi

Reply:
Zapravo, fali nam i provera `isUserCustomer`. Ako nije, postavi dugme na disabled. Ima u `util/auth` pomocna funkcija koja to proverava